### PR TITLE
Documetation updates

### PR DESCRIPTION
**Pull request by joe-singleton-itdev**
Update documentation for release 4:

- Removed 'sudo' from all examples because it is no longer necessary
- Updated installation instructions
- Updated clustering instructions
- Removed some bits that are no longer relevant
- Added user management stuff in

### CODE REVIEW

**Comment by itdevreviewboard** [comment]
Can one of the MCVirt admins verify this patch?

**Comment by MatthewJohn** [comment]
ok to test

**Comment by itdevreviewboard** [comment]
Build triggered. sha1 is merged.

**Comment by itdevreviewboard** [comment]
Build started sha1 is merged.

**Comment by itdevreviewboard** [comment]
Build Failure!!

**Comment by MatthewJohn** [comment]
ok to test

**Comment by itdevreviewboard** [comment]
CI build triggered.

**Comment by itdevreviewboard** [comment]
CI build started.

**Comment by itdevreviewboard** [comment]
Build Failure!!

**Comment by MatthewJohn** [comment]
ok to test

**Comment by itdevreviewboard** [comment]
CI build triggered.

**Comment by itdevreviewboard** [comment]
CI build started.

**Comment by itdevreviewboard** [comment]
Build Failure!!

File: Documentation/Cluster.rst [hunk 1]
```diff
@@ -75,30 +79,30 @@ Removing a node from the cluster
...
 Get Cluster information
 -----------------------
 
 * In order to view status information about the cluster, use the 'info' parameter for MCVirt, without specifying a VM name::
 
-    sudo mcvirt info
+    mcvirt info
 
 
 Off-line migration
```
Comment (MatthewJohn) [inline Documentation/Cluster.rst]: Sorry to be a pain - would you be able to rename this to 'Virtual Machine Migration' and stick a note about the '--online' flag that can perform online migration

File: Documentation/Cluster.rst [hunk 2]
```diff
@@ -50,18 +49,23 @@ Joining the node to the cluster
...
-    sudo mcvirt cluster add-node --node <Remote Node Name> --ip-address <Remote Cluster IP Address>
-    
 
-2. A prompt for the root password of the remote node will be presented.
+    mcvirt cluster add-node --connect-string <connect string>
+
+where ``<connect string>`` is the string printed out in step 1.
+
+
 3. The local node will connect to the remote node, ensure it is suitable as a remote node, setup authentication between the nodes and copy the local permissions/network/virtual machine configurations to the remote node.
```
Comment (MatthewJohn) [inline Documentation/Cluster.rst]: Could you put a note mentioning that all data on the remote node (e.g. VMs, users, permissions etc) will be removed whilst it is added to a cluster?

File: Documentation/Configuration.rst [hunk 3]
```diff
@@ -35,21 +35,29 @@ Where `<Physical interface>` is the name of the interface that the bridge should
...
 
-    sudo mcvirt node --set-vm-vg <Volume Group>
+    mcvirt node --set-vm-vg <Volume Group>
 
 * The cluster IP address must be configured if the node will be used in a cluster (See the `Cluster documentation <Cluster.rst>`_)::
 
-    sudo mcvirt node --set-ip-address <Cluster IP Address>
+    mcvirt node --set-ip-address <Cluster IP Address>
+
+* If the node will be used in a cluster, the hosts file at ``/etc/hosts`` must edited by changing the line::
```
Comment (MatthewJohn) [inline Documentation/Configuration.rst]: This needs to be done even if the node isn't added to a cluster, as it is required for the RPC client to connect to the local machine

File: Documentation/ControllingVMs.rst [hunk 4]
```diff
@@ -39,12 +39,12 @@ Reset VM
...
 
-* Use virsh to reset VMs:
+* Use the MCVirt utility to reset VMs:
 
   ::
-    
-    virsh reset <VM Name>
-    
+
+    mcvirt reset <VM Name>
```
Comment (MatthewJohn) [inline Documentation/ControllingVMs.rst]: Thanks for picking this up :congratulations:

File: Documentation/Installation.rst [hunk 5]
```diff
@@ -27,17 +27,9 @@ To install the package, run::
...
-* If a user is to be able to use MCVirt and does not already have permission to run commands using sudo, the following sudoers rule can be used::
-
-    example_username ALL=(ALL) /usr/bin/mcvirt
-    %example_group ALL=(ALL) /usr/bin/mcvirt
-
-Additionally, ``NOPASSWD:`` can be used to allow users to run MCVirt without having to re-enter their password::
+MCVirt uses a customised version of `Pyro <https://pythonhosted.org/Pyro4/>`_, which can be installed by running::
 
-    example_user ALL=(ALL) NOPASSWD: /usr/bin/mcvirt
+$ git clone https://github.com/MatthewJohn/Pyro4
```
Comment (MatthewJohn) [inline Documentation/Installation.rst]: Possibly mention installing python-pip package?

**Comment by MatthewJohn** [comment]
Other than the above comments, the changes look really good :happy

File: Documentation/Installation.rst (hunk 5 above)
Comment (joe-singleton-itdev) [inline Documentation/Installation.rst]: See line 35!

**Comment by MatthewJohn** [comment]
ok to test

**Comment by joe-singleton-itdev** [comment]
Ship it :shipit:

**Comment by MatthewJohn** [comment]
ship it! :shipit:

**Pull request merged**